### ISSUE TITLE
Catch code outside of class or define block.

### ISSUE THREAD

**Issue by traylenator** [rodjek/puppet-lint#220]
Would be good to catch include statements or anything outside of a class or define.
e.g. 

```puppet
class foo::bar {
  file{'/etc/passwd':}
}
include ('mymodule')
```
or the similar
```puppet
define foo::bar (){
  file{'/etc/passwd':}
}
include ('mymodule')
```

both or these are bad practice but they also happen to trigger a bug.

**Comment by rnelson0** [comment]
Lee Lowder has created a plugin to handle this check at https://github.com/llowder/puppet-lint-resource_outside_class-check.
